### DISCUSSION TITLE
Provide more slots on all basic components

### DISCUSSION THREAD

**Discussion by Acetyld** [primefaces/community#1217] [PrimeVue]
I already made 2/3 discussions on missing slots and i also saw some others

In past 3/4 months the thing that came coming back to me was missing slots on many components

for example the tree select does not have all slots the tree component has, table has not footer slot (internal footer of scroll content), dropdown has no input slot, calendar has no time slot, radio button has no button slot

etc, etc. Ofcourse primevue is not as flexible as a library like  shadcn or radix but would be lovely to be more flexible in extending existing components for other end goals and/or basic changes that are hard to do without a wrapper or custom solution or pull request ( currently for example i dont use tree select but i use a readonly TextInput inside a FloatingVue dropdown menu (the one from primevue was missing some stuff) together with the Tree component to get all slots and more customization)

Keep in mind!! No rand i love prime vue, but this is the only thing that keeps coming back while using it

**Comment by ipimpat** [comment]
Agree, the lack of slots is very frustrating
